src/parameter/ParameterWidget.cc

```diff
@@ -388,6 +396,24 @@ void ParameterWidget::updateParameterSet(std::string setName)
 			this->comboBoxPreset->addItem(s, QVariant(s));
 			this->comboBoxPreset->setCurrentIndex(this->comboBoxPreset->findText(s));
 		}
-		writeParameterSet(this->jsonFile);
+		writeParameterSets();
 	}
 }
+
+void ParameterWidget::writeParameterSets()
+{
+	if(this->unreadableFileExists){
+		bool ok = true;
+		QMessageBox msgBox;
+		msgBox.setText(_("Saving presets"));
+		msgBox.setInformativeText(QString(_("%1 was found, but was unreadble. Do you want to overwrite %1?")).arg(QString::fromStdString(jsonFile)));
+		msgBox.setStandardButtons(QMessageBox::Save | QMessageBox::Cancel);
+		msgBox.setDefaultButton(QMessageBox::Cancel);
+		int ret = msgBox.exec();
+
+		if (ret == QMessageBox::Cancel) {
+			return;
+		}
```

Comment:
You could probably make the above a tiny bit cleaner by doing:
`if (msgBox.exec() == QMessageBox::Cancel) return;`